### PR TITLE
👩‍🌾 Foxy and Rolling supported, Galactic staged

### PR DESCRIPTION
**Pull request by chapulina**
* Make Foxy and Rolling supported
* Set Galactic as the next one coming up

### CODE REVIEW

File: jenkins-scripts/dsl/_configs_/Globals.groovy
Comment (mjcarroll) [inline jenkins-scripts/dsl/_configs_/Globals.groovy:113]: Should this be 'rolling'?

**Comment by scpeters** [comment]
I'll defer to @mjcarroll's review of this

File: jenkins-scripts/dsl/_configs_/Globals.groovy
Comment (chapulina) [reply jenkins-scripts/dsl/_configs_/Globals.groovy:113]: Yeah maybe this entire logic could be a bit updated to account for the new rolling paradigm. The reason I didn't put `rolling` here is that we probably want rolling install jobs. See


https://github.com/ignition-tooling/release-tools/blob/c5045684ef7915bc29c6573ad4a624de5b402084/jenkins-scripts/dsl/gazebo_ros_pkgs.dsl#L165-L168

Comment (mjcarroll) [reply jenkins-scripts/dsl/_configs_/Globals.groovy:113]: Yes, it probably makes sense to have install jobs.  I think now generally that the ordering may be a bit inverted, in that we will have packages for rolling, that will eventually become the packages for galactic and further.

Comment (chapulina) [reply jenkins-scripts/dsl/_configs_/Globals.groovy:113]: Cool, so I think `galactic` here is ok for now?

Comment (j-rivero) [reply jenkins-scripts/dsl/_configs_/Globals.groovy:113]: agree with Louise. The logic probably needs to be updated to match what currently is the "developement_distro".

**Review by j-rivero — APPROVED** [review]

**Comment by j-rivero** [comment]
Doing a couple of tests before merging:
 *  [Build Status](https://build.osrfoundation.org/job/ros2_gazebo_pkgs-ci-default_foxy-focal-amd64/21/)
 * [Build Status](https://build.osrfoundation.org/job/ros2_gazebo_pkgs-ci-default_rolling-focal-amd64/2/)

File: jenkins-scripts/dsl/_configs_/Globals.groovy
Comment (mjcarroll) [reply jenkins-scripts/dsl/_configs_/Globals.groovy:113]: :+1:

**Review by mjcarroll — APPROVED** [review]

**Comment by j-rivero** [comment]
> Doing a couple of tests before merging:

I think we need to take care of defining the right branches for foxy and rolling somewhere. This change would create `gazebo_ros_pkgs` jobs that will fail.

**Comment by chapulina** [comment]
> we need to take care of defining the right branches for foxy and rolling somewhere.

They should both be the `ros2` branch at the moment. I guess it is looking for `foxy` and `rolling`

**Comment by j-rivero** [comment]
After a5dd2ca 

 * rolling [Build Status](https://build.osrfoundation.org/job/ros2_gazebo_pkgs-ci-default_rolling-focal-amd64/3/)
 * foxy [Build Status](https://build.osrfoundation.org/job/ros2_gazebo_pkgs-ci-default_foxy-focal-amd64/22/)

Seems like there are problems with the tests finding plugins but the compilation mostly work and I think the changes in this PR are good to go.

**Pull request merged**
